sdks/java/extensions/sql/datacatalog/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/datacatalog/DataCatalogTableProvider.java

```diff
@@ -98,9 +98,8 @@ public void dropTable(String tableName) {
   }
 
   @Override
-  public @Nullable Table getTable(String tableNamePart) {
-    throw new UnsupportedOperationException(
```

Comment:
Where does this exception get swallowed and turned into a null? Did you figure that out?

Reply:
Yes. See next comment.